model/types.go

```diff
@@ -284,7 +284,7 @@ type AccessInfo struct {
 type BlockDeviceAccessInfo struct {
 	AccessProtocol string   `json:"access_protocol,omitempty"`
 	TargetNames    []string `json:"target_names,omitempty"`
-	LunID          int32    `json:"lun_id,omitempty"`
+	LunID          string   `json:"lun_id,omitempty"`
```

Comment:
I think the type should directly state what the expectation is.  In this case, it should be []int32.

Reply:
Fixed in the recent update.